src/Client/Web/Bit.Client.Web.BlazorUI/Components/DropDown/BitDropDown.razor

```diff
@@ -75,65 +75,58 @@
          tabindex="0"
          role="listbox"
          aria-labelledby="@DropdownLabelId">
-        @foreach ((BitDropDownItem item, Int32 index) in Items.Select((item, index) => (item, index)))
+        @foreach ((BitDropDownItem item, int index) in Items.Select((item, index) => (item, index)))
         {
             if (item.ItemType == BitDropDownItemType.Header)
             {
-                <div class="@RootElementClass-head-@VisualClassRegistrar()"
-             id="@($"{DropDownId}{item.Text?.ToLower()}Header")">
+                <div class="@RootElementClass-head-@VisualClassRegistrar()" id="@($"{DropDownId}{item.Text?.ToLower()}Header")">
                     @item.Text
                 </div>
             }
             else if (item.ItemType == BitDropDownItemType.Normal)
             {
                 @if (IsMultiSelect)
                 {
-                    <BitCheckbox Class="@(item.IsSelected ? $"{RootElementClass}-slc-{VisualClassRegistrar()}" :"")"
-                     Style="@(item.IsHidden ? "display: none" : "")"
-                     IsChecked="item.IsSelected"
-                     IsEnabled="item.IsEnabled"
-                     OnClick="(e) => HandleItemClick(item)"
-                     id="@($"{DropDownId}-list{index}")"
-                     type="checkbox"
-                     role="option"
-                     data-is-focusable="@(item.IsEnabled ? "true" : "false")"
-                     disabled=@(!item.IsEnabled)
-                     data-index="@index"
-                     aria-disabled="@(!item.IsEnabled)"
-                     aria-selected="@(item.IsSelected ? "true" : "false")"
-                     aria-setsize="@NormalDropDownItems.Count"
-                     aria-posinset="@GetItemPosInSet(item)"
-                     tabindex="@(item.IsEnabled ? 0 : -1)"
-                     title="@item.Title"
-                     AriaLabel="@item.AriaLabel">
```

Comment:
why these attributes have been removed?

Reply:
I think it's ok, these were attributes of the BitCheckBox which was removed completely.